### ISSUE TITLE
Squeezebox server icon

### ISSUE THREAD

**Issue by jeanpaulrh** [walkxcode/dashboard-icons#295]
### Type

Icon request

### Info

Hi, I would like to have a logitech squeezebox server icon added.
Thanks!

### Icon(s)

![squeezeboxrpc](https://user-images.githubusercontent.com/4814843/222349416-add69486-7613-4702-9ecd-2531cbd02c72.png)

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/squeezebox-server.png
